Dockerfile

```diff
@@ -1,12 +1,14 @@
+FROM golang:alpine
+RUN apk --no-cache add git make
+RUN go get github.com/runatlantis/atlantis/
```

Comment:
I don't think this will work because it's going to pull master when instead we want to use the codebase that exists whenever CI is running. It usually would be the same but I don't think it makes sense to complicate this.